vividus-plugin-web-app/src/main/java/ru/yandex/qatools/ashot/shooting/DebuggingViewportPastingDecorator.java

```diff
@@ -39,6 +39,8 @@
  */
 // CHECKSTYLE:OFF
 public class DebuggingViewportPastingDecorator extends ShootingDecorator {
+    private static final String SCROLL_Y = 
+            ".scrollTo({\"top\":arguments[0], \"left\": 0, \"behavior\":\"instant\"}); return [];";
```

Comment:
IE doesn't support this API and only the latest version (24.03.2020) of Safari supports it: https://caniuse.com/#search=scrollTo

Reply:
Do you agree to use it for the browsers supporting the API?

Reply:
yes, sounds reasonable